engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -95,6 +97,19 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
+    @Override
+    public void setUri(String nodeUri) {
```

Comment:
The original interface that this method is declared in (Node) is a public interface. Which means either I'll have to change it from `public interface` to `interface`, or we'll have to live with this.
Your call.